runtime/src/snapshot_utils.rs

```diff
@@ -2988,6 +2988,27 @@ pub fn purge_old_bank_snapshots(
     );
 }
 
+/// At startup, purge old (i.e. unusable) bank snapshots
+///
+/// Only a single bank snapshot could be needed at startup (when using fast boot), so
+/// retain the highest bank snapshot "post", and purge the rest.
+pub fn purge_old_bank_snapshots_at_startup(bank_snapshots_dir: impl AsRef<Path>) {
```

Comment:
hesistent on the naming here, we're using it at startup now, but actual code isn't really restricted that way. What do you think?